### ISSUE TITLE
Add support for OpenBMC

### ISSUE THREAD

**Issue by luke-jarymowycz** [bmc-toolbox/bmclib#235]
Must include:
power functions
user credential operations

**Comment by joelrebel** [comment]
@zevweiss does the OpenBMC we're shipping expose redfish endpoints? atleast it seemed like it based on some of your work here https://github.com/bmc-toolbox/bmclib/pull/167/files

**Comment by zevweiss** [comment]
Yeah, it's got both redfish and ipmi available, though there may be gaps in their respective implementations regarding certain operations -- I'm pretty sure power control should be available via both, but I've never tested user/password management operations.
